### ISSUE TITLE
module 'autokeras' has no attribute 'StructuredDataRegressor'

### ISSUE THREAD

**Issue by takuya2816** [keras-team/autokeras#888]
### Problem
When I tried to use 'StructuredDataRegressor', the error happens.

```
import autokeras as ak
auto_sdr = ak.StructuredDataRegressor()
auto_sdr.fit(train_X, train_y)
```
>  AttributeError: 
> module 'autokeras' has no attribute 'StructuredDataRegressor'

### Environment
OS: Windows10
Python: 3.6.7
tensorflow: 2.0.0
autokeras: 1.0.0b0

### Check the directory of autokeras package
There is not StructuredData file at the directory of autokeras package.
(However, there are image and text file)

Is there any way to solve this?

**Comment by haifeng-jin** [comment]
@takuya2816 We cannot reproduce the error. It might because the python interpreter you use is not the one you installed the 1.0 version of AutoKeras.
If you are running your program with command `python3`.
You can try this `python3 -m pip freeze | grep autokeras`. to see the version.

**Comment by haifeng-jin** [comment]
This is because the environment recognize the ak package as the one in the tests folder. directly install autokeras instead of cloning the repo would solve the problem.

**Comment by iszee** [comment]
i'm having the same issue in conda env and local env

**Comment by haifeng-jin** [comment]
This issue is resolved in the latest master.
We have renamed the tests/autokeras to tests/unittests.
It must be something else.

**Comment by Ayad0net** [comment]
> If you have in your folder script with the name "autokeras.py" rename it.

Thanks 🚀

**Comment by Vallabh100** [comment]
solved using autokeras version 1.0.20
